tools/e2etesting/GetSecrets.ps1

```diff
@@ -16,4 +28,25 @@ foreach ($secret in $secrets) {
 	$values[$secret.Name.ToUpperInvariant().Replace("-", "_")] = $secretValueText
 }
 
-$values | ConvertTo-Json -Depth 10
\ No newline at end of file
+# Find the full path of the launchSettings.json file
+$folderName = "e2e-tests"
+$currentPath = (Get-Location).Path
+$path = Get-ChildItem -Path $currentPath $folderName -Recurse 
```

Comment:
Question: why do you need `-Recurse` here while on line `37` it is not present when performing the same search ?

Reply:
First we perform the forward search. For example if you started the script from C:\ it needs to go to the repository and check what's inside, so it would be C:\Industrial-IoT\e2e-tests.
If that doesn't work we can assume that we are already inside of the repository and we have to go back. For example if I start the script from C:\Industrial-IoT\tools\e2etesting it will go two folders back and find e2e-tests there.